packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/utility-nav.twig

```diff
@@ -1,4 +1,6 @@
 <div class="ma__utility-nav js-util-nav">
+  {# Unable to use aria for labeling since inconsistent support for the tags by screen readers as of Feb. 22, 2023. #}
+  <span id="utility-nav-label" class="ma__visually-hidden">Language options and quick access links</span>

```

Comment:
During the testing the changes, I found out that JAWS doesn't support `aria-label` for static elements.  So, tried `aria-labelledby`, then NVDA doesn't support it on static elements.  Both cannot be used because VoiceOver announces them.  I hate to leave out the semantic association between the elements and their labels, but in this case, letting the reading order handle their association seem to be a reasonable approach.

Reply:
It's good to know that neither `aria-label` nor `aria-labelledby` works consistently with lists. I was aware of that before. Iagree in this case there's no better option if we want to label the list then.